src/main/java/io/moderne/organizations/OrganizationStructureService.java

```diff
@@ -14,93 +12,34 @@
 import java.io.*;
 import java.nio.file.Path;
 import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
 
 /**
  * This service could also directly call your SCM api to determine the available repositories
  */
 @Service
 public class OrganizationStructureService {
     private static final String DEFAULT_REPOS_CSV = "repos.csv";
-    private static final String NAME_MAPPING = "id-mapping.txt";
-    private static final String COMMIT_OPTIONS = "commitOptions.txt";
     private static final Logger log = LoggerFactory.getLogger(OrganizationStructureService.class.getName());
 
-    private final ScmConfiguration scmConfiguration;
     private final GitRemote.Parser gitRemoteParser;
 
     @Nullable
     private final Path reposCsvPath;
 
     public OrganizationStructureService(ModerneConfiguration moderneConfiguration, GitRemote.Parser gitRemoteParser) {
         this.gitRemoteParser = gitRemoteParser;
-        this.scmConfiguration = moderneConfiguration.getScm();
         this.reposCsvPath = moderneConfiguration.getReposCsvPath();
     }
 
     OrganizationTree readOrganizationStructure() {
         LinkedHashMap<String, OrganizationRepositories> organizations = new LinkedHashMap<>();
-        Set<RepositoryInput> allRepositories = new LinkedHashSet<>();
-
-        final Map<String, String> idToNameMapping = readIdToNameMapping();
-
-        CsvMapper csvMapper = new CsvMapper();
-        CsvSchema bootstrapSchema = CsvSchema.emptySchema()
-                .withComments()
-                .withHeader();
-
-        try (InputStream reposCsvInputStream = loadReposCsvInputStream();
-             InputStream commitOptionsInputStream = loadCommitOptionsInputStream();
-             MappingIterator<ReposCsvRow> iterator = csvMapper.readerFor(ReposCsvRow.class)
-                     .with(bootstrapSchema)
-                     .readValues(reposCsvInputStream)) {
-            CommitOptions commitOptions = CommitOptions.fromInputStream(commitOptionsInputStream);
-            while (iterator.hasNextValue()) {
-                ReposCsvRow row = iterator.nextValue();
-                String cloneUrl = row.getCloneUrl();
-                RepositoryInput repository = determineRepository(cloneUrl, row.getBranch());
-                if (repository == null) {
-                    if (scmConfiguration.isAllowMissingScmOrigins()) {
-                        log.warn("No scm origin found for %s. Consider adding it to scm-origins.txt".formatted(cloneUrl));
-                        continue;
-                    } else {
-                        throw new IllegalStateException("No scm origin found for %s. Add it to moderne.scm.repositories or set moderne.scm.allow-missing-scm-origins to true in application.yaml".formatted(cloneUrl));
-                    }
-                }
```

Comment:
Did not port this logic as `determineRepository` never returns null so this code path is never executed.

Source for reference:

```
RepositoryInput determineRepository(String cloneUrl, String branch) {
        GitRemote gitRemote = gitRemoteParser.parse(cloneUrl);
        return new RepositoryInput(gitRemote.getPath(), gitRemote.getOrigin(), branch);
}
```